packages/mwp-app-render/src/components/Dom.jsx

```diff
@@ -36,9 +38,19 @@ const DOM = props => {
 		initialState = {},
 		scripts,
 		cssLinks,
+		userAgent,
 	} = props;
 
-	const htmlLang = initialState.config.requestLanguage.split('-')[0];
+	const localeCode = initialState.config.requestLanguage;
+	const htmlLang = localeCode.split('-')[0];
+
+	/**
+	 * Add polyfill.io script if needed
+	 */
+	const polyfill = getPolyfill(userAgent, localeCode);
+	if(polyfill) {
+		scripts.push(polyfill);
```

Comment:
@mmcgahan I went with this approach, but we want the polyfill JS to load before the app code, and this doesn't achieve that since it sticks it to the end of the array. I think I'd prefer `unshift` here.

Reply:
ah, yeah, totally mean `unshift` 👍